src/mol-plugin-ui/skin/base/components/viewport.scss

```diff
@@ -25,14 +25,14 @@
     bottom: 0;
     -webkit-user-select: none;
     user-select: none;
-    -webkit-tap-highlight-color: rgba(0,0,0,0);
+    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
     -webkit-touch-callout: none;
     touch-action: manipulation;
 
-    > canvas {
+    >canvas {
```

Comment:
Please fix the formatting in this CSS file (highlighted by the change view)

Reply:
Fixed (for some reason my autoformat dislikes the space after >).